src/Sake/Program.cs

```diff
@@ -36,12 +37,21 @@ static void Main(string[] args)
 
         static IContainer CreateContainer(ISakeSettings settings)
         {
+            var isTeamCity = !string.IsNullOrEmpty(Environment.GetEnvironmentVariable("TEAMCITY_VERSION"));

```

Comment:
Move this check into `_use-teamcity.shade`, and the default service doesn't need to be if()'ed